### PR TITLE
integration: use Range to wait for reboot in quota tests

### PR DESCRIPTION
**Pull request by heyitsanthony**
Proxy client layer ignores call options so Put is always FailFast.

Tried adding WithCallOptions as a client OpOption to do CallOption passthrough but the change set was larger than what seemed reasonable to support essentially a testing path.

### CODE REVIEW

File: integration/v3_grpc_test.go [hunk 1]
```diff
@@ -598,9 +598,13 @@ func TestV3StorageQuotaAPI(t *testing.T) {
 
 	key := []byte("abc")
 
+	// wait for server to reboot
+	if _, err := kvc.Range(context.TODO(), &pb.RangeRequest{Key: key}, grpc.FailFast(false)); err != nil {
```
Comment (xiang90) [inline integration/v3_grpc_test.go]: this relies on the knowledge that proxy uses etcd.client, which will do range with failfast=true no matter what. Right?

Comment (heyitsanthony) [inline integration/v3_grpc_test.go]: for the proxy tests to pass it expects the client to do failfast=false for range; this is suddenly breaking the regular test though, so clearly needs more thought...

**Comment by heyitsanthony** [comment]
all fixed /cc @xiang90

**Comment by xiang90** [comment]
lgtm

**Pull request merged**